docs/user/access-control.md

```diff
@@ -75,54 +106,158 @@ kubectl create -n $KUBEAPPS_NAMESPACE rolebinding example-kubeapps-repositories-
 
 Service Brokers, Classes and Plans in the Service Catalog are cluster-scoped
 resources, but Service Instances and Bindings can be restricted to a namespace.
-Kubeapps defines two roles (`kubeapps-service-catalog-browse` and
+We'll need to define two roles (`kubeapps-service-catalog-browse` and
 `kubeapps-service-catalog-read`) to separate the roles required to view Service
 Instances and Bindings so that they can be applied to desired namespaces.
 
-In order to list and view Service Instances in a namespace, apply the
+In order to list and view Service Instances in a namespace, we'll create the
 `kubeapps-service-catalog-browse` ClusterRole in all namespaces and the
 `kubeapps-service-catalog-read` in the desired namespace.
 
 ```
+kubectl apply -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kubeapps-service-catalog-browse
+rules:
+- apiGroups:
+  - servicecatalog.k8s.io
+  resources:
+  - clusterservicebrokers
+  - clusterserviceclasses
+  - clusterserviceplans
+  verbs:
+  - list
+EOF
```

Comment:
I definitely think that we need to extract these yamls to external files since these documentation is becoming too dense.

In the meantime I'd add spaces and comments between the different steps.